database/2021_11_02_202021_update_wallets_uuid_table.php

```diff
@@ -40,7 +40,12 @@ public function up(): void
 
     public function down(): void
     {
-        Schema::dropColumns($this->table(), ['uuid']);
+        Schema::table($this->table(), function (Blueprint $table) {
```

Comment:
If the migration is marked as completed, the column should be there.

The migration first creates the column and then fills it. Rollback won't help here.

https://bavix.github.io/laravel-wallet/guide/introduction/upgrade.html#_6-2-4-%E2%86%92-7-x-x

I described that for mysql it is better to mark the migration as completed and perform it manually.
For postgres use CONCURRENTLY.

Reply:
The issue I've encountered was during a fresh install of laravel with latest package version.

If you do migrate and rollback for some reason it cause an issue, the same with Pestphp that uses `RefreshDatabase` on their testing.

Reply:
It does not affect the work globally. Good. Thanks for the PR